tests/conftest.py

```diff
@@ -206,11 +205,12 @@ def cmd(cmd, cwd=None, stderr=None):
     # stdout from cmd is captured and returned. The command is run in
     # a python subprocess so that program-level setup and teardown
     # happen fresh.
-
-    cmdlst = shlex.split('west ' + cmd)
-    print('running:', cmdlst)
+    cmd = 'west ' + cmd
+    if platform.system() != 'Windows':
```

Comment:
We typically use `sys.platform`, maybe use that for consistency

Reply:
Why is this needed btw? We don't usually make this difference in normal invokations of `check_output` et al.

Reply:
> We typically use sys.platform, maybe use that for consistency

Really? We use platform.system() in configuration.py ,which is what this is testing. I can make that change, though.

> Why is this needed btw? We don't usually make this difference in normal invokations of check_output et al.

It's breaking the tests because shlex splits by POSIX rules, which isn't right for windows shells and led to bad results when applied to paths. Example:

```
In [2]: shlex.split('west foo bar\\baz')
Out[2]: ['west', 'foo', 'barbaz']
```

Reply:
Right, I see. I don't have a strong preference either way, just wanted to keep it consistent. I didn't know we used `platform.system()` in `configuration.py`, but I've always seen `sys.platform` elsewhere, like the `tox.ini` or the `requirements.txt`.

Reply:
5 years later, dropping `shlex.split()` in #730